README.md

```
@@ -34,4 +34,8 @@ With [DAVe](https://opensource.muenchen.de/software/dave.html), traffic counts c
 
 ## Roadmap
 
-![DAVe Roadmap](img/DAVe_Roapmap.png)
+<img src="img/DAVe_Roapmap.png" width="850">
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)